tjenestebuss-integrasjon/build.gradle.kts

```diff
@@ -11,12 +11,19 @@ plugins {
 group = "no.nav.pensjon.brev.tjenestebuss"
 version = "0.0.1"
 
+repositories {
+    maven {
+        url = uri("https://github-package-registry-mirror.gc.nav.no/cached/maven-release")
+    }
+}
+
 application {
 	mainClass.set("no.nav.pensjon.brev.tjenestebuss.tjenestebussintegrasjon.TjenestebussIntegrasjonApplicationKt")
 }
 
-val cxfVersion = "3.6.3"
-val tjenestespesifikasjonerVersion = "1.858e92e"
+// Før du oppdaterer cxf-versjonen, sjekk readme-fila for denne modulen.
+val cxfVersion = "4.1.3"
```

Comment:
Har du manuelt testet at denne versjonen fungerer mot den tjenesten vi bruker? Det er en særegen funksjon vi kaller på som teknisk sett ikke er helt en del av bussen.

Reply:
Testa lokalt i docker-compose, det fungerte heilt fint, så deploya til q2 no. Ser ut til å funke heilt fint der også, men har foreløpig berre testa å søke på organisasjonsnavn, men det ser jo ut til å vera samme endepunkt for alt.

Reply:
Da fungerer det nok. Jeg ville bare være sikker, da jeg mener å huske grunnen til at vi har holdt tilbake versjonen var fordi det ikke fungerte mot den tjenesten.

Reply:
Ja, det var ikkje akkurat rett fram å oppgradere her, men summen av desse endringane gjorde susen.